cmds/core/brctl/brctl.go

```diff
@@ -0,0 +1,192 @@
+// brctl setbridgeprio <bridge> <priority>		sets the bridge's priority to <priority>
+// brctl setfd <bridge> <time>					sets the bridge's 'bridge forward delay' to <time> seconds
+// brctl sethello <bridge> <time>				sets the bridge's 'bridge hello time' to <time> seconds
+// brctl setmaxage <bridge> <time>				sets the bridge's 'maximum message age' to <time> seconds.
+// brctl setpathcost <bridge> <port> <cost>		sets the port cost of the port <port> to <cost>. This is a dimensionless metric
+// brctl setportprio <bridge> <port> <priority>	sets the port <port>'s priority to <priority>
+// brctl hairpin <bridge> <port> <state>		enable/disable hairpin mode on the port <port> of the bridge <bridge>
+
+// Busybox Implementation: https://elixir.bootlin.com/busybox/latest/source/networking/brctl.c
+// Kernel Implementation: https://mirrors.edge.kernel.org/pub/linux/utils/net/bridge-utils/
+// Author: Leon Gross (leon.gross@9elements.com)
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/brctl"
+)
+
+// cli
+const usage = `
+Usage: brctl [commands]
+commands:
+
+INSTANCES:
+brctl addbr <name>		creates a new instance of the ethernet bridge
+brctl delbr <name>		deletes the instance <name> of the ethernet bridge
+brctl show			show current instance(s) of the ethernet bridge
+
+PORTS:
+brctl addif <brname> <ifname>	will make the interface <ifname> a port of the bridge <brname>
+brctl delif <brname> <ifname>	will detach the interface <ifname> from the bridge <brname>
+brctl show <brname>		will show some information on the bridge and its attached ports
+
+AGEING:
+brctl showmacs <brname> 		shows a list of learned MAC addresses for this bridge
+brctl setageingtime <brname> <time>	sets the ethernet (MAC) address ageing time, in seconds [OPT]
+brctl setgcint <brname> <time> 		sets the garbage collection interval for the bridge <brname> to <time> seconds [OPT]
+
+SPANNING TREE PROTOCOL (IEEE 802.1d):
+brctl stp <bridge> <state>			controls this bridge instance's participation in the spanning tree protocol.
+brctl setbridgeprio <bridge> <priority>		sets the bridge's priority to <priority>
+brctl setfd <bridge> <time>			sets the bridge's 'bridge forward delay' to <time> seconds
+brctl sethello <bridge> <time>			sets the bridge's 'bridge hello time' to <time> seconds
+brctl setmaxage <bridge> <time>			sets the bridge's 'maximum message age' to <time> seconds.
+brctl setpathcost <bridge> <port> <cost>	sets the port cost of the port <port> to <cost>. This is a dimensionless metric
+brctl setportprio <bridge> <port> <priority>	sets the port <port>'s priority to <priority>
+brctl hairpin <bridge> <port> <state>		enable/disable hairpin mode on the port <port> of the bridge <bridge>
+`
+
+func run(out io.Writer, argv []string) error {
+	var err error
+	command := argv[0]
+	args := argv[1:]
+
+	switch command {
+	case "addbr":
```

Comment:
I have a hard time validating the error. IIRC we had the same error and it was related to missing kernel options being built in. Since the CI works, I guess this is no longer an issue? 

Please let me know if this is the case or not.

Reply:
PR https://github.com/u-root/u-root/pull/2995 shows which kernel features are required for brctl to work. If you want to run it locally but get the `package not installed` error you could check the kernel config of your running host system for the features (e.g. /proc/config.gz, /boot/config*, ...)

Reply:
I was using setup here as is, https://github.com/linuxboot/linuxboot/tree/main/mainboards/qemu/x86_64

It was a while back. Now tests pass. I'll give it another try.